src/sql/azdata.proposed.d.ts

```diff
@@ -44,6 +44,12 @@ declare module 'azdata' {
 		export interface IKernelChangedArgs {
 			nbKernelAlias?: string
 		}
+
+		export interface IExecuteResult {
```

Comment:
Should this live in proposed?

Reply:
@aaomidi mentioned changes to azdata should be done in proposed